gradio/components/button.py

```diff
@@ -29,8 +30,8 @@ def __init__(
         every: Timer | float | None = None,
         inputs: Component | Sequence[Component] | set[Component] | None = None,
         variant: Literal["primary", "secondary", "stop", "huggingface"] = "secondary",
-        size: Literal["sm", "lg"] | None = None,
-        icon: str | None = None,
+        size: Literal["sm", "md", "lg"] | None = None,
```

Comment:
I added a "md"-sized button as that was ideal for this UI and I think it could be more broadly useful.

Reply:
Made the same change to all of the subclasses of Button below